src/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { createStore, applyMiddleware } from 'redux';
+import {ParallaxProvider} from 'react-scroll-parallax';
+import  apiMiddleware  from './redux/middlewares/api';
+
+import './index.css';
+import reducer from './redux/reducers/reducer'
+import App from './App';
+
+import * as serviceWorker from './serviceWorker';
+
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || window.compose;
+
+const store = createStore(
```

Comment:
Perfect example on when to start splitting codebase into files. You can set up your store in a `store.js` file that you can import here as its configuration is starting to grow.